README.md

```diff
@@ -59,13 +59,13 @@ Brightness Controller changes Red, Green and Blue color ratios  in the screen th
 
 We work on this in our spare time, so can not really promise when the v3 will be released. The current version available is stable and should work as intended.
 
-## Dependencies
```

Comment:
Hmm, interesting.
Can you elaborate on how you are differentiating between requirements and dependencies?

Reply:
The two terms are similar, but "requirements" are easier to understand in this context. It's "required" to have Python installed. Dependency is more like a similar programming package (e.g. PyPI package), while requirement is more general.

Also I wouldn't call `python3` a "dependency" in any case. :)